fips.go

```diff
@@ -0,0 +1,13 @@
+package notary
+
+import "os"
+
+// FIPSEnvVar is the name of the environment variable that is being used to switch
+// between FIPS and non-FIPS mode
+const FIPSEnvVar = "GOFIPS"
+
+// FIPSEnabled returns true if environment variable `GOFIPS` has been set to enable
+// FIPS mode
+func FIPSEnabled() bool {
+	return os.Getenv(FIPSEnvVar) != ""
```

Comment:
Ah, I personally think it should. But I tried to replicate what @FiloSottile did in his library. Basically, 0 or "false" are true values as of now in GCS. If you strongly feel it should be updated, then I'll update in here as well as send a PR there.